src/routes/LandingPage/SponsorSection/index.scss

```diff
@@ -0,0 +1,48 @@
+.sponsor-section {
+  .sponsor-head {
+    display: flex;
+    margin-top: 5%;
+    margin-right: 5%;
+    margin-left: 5%;   
+    flex-grow: 1;
+    flex-shrink: 1;
+    justify-content:space-between;
+    overflow-inline: hidden;
+    
+    .sponsor-button {
+      color: white;
+      align-self: flex-start;
+      background-color: rgba(130, 52, 218, 1);
```

Comment:
is this same colour as one of the purples in `global.scss` if so, we could import that

Reply:
are u fucking stupid? u re-requested a review but this is stil here? are u happy with ur work? Is this the standards u hold for urself? or are u just so fucking stupid that u managed to miss this completely
I dont know whats worse, u being a lazy sacck of shit who thinks this deserves to be merged in or, ur so stupid that somehow, u dont even know why this is bad